docs/src/main/sphinx/release/release-408.md

```diff
@@ -0,0 +1,55 @@
+# Release 408 (22 Feb 2023)
+
+## General
+
+* Fix query failure when performing a remote partitioning exchange on a
+  [structural type](structural-data-types) that contains `NULL` elements. ({issue}`16140`)
+
+## CLI
+
+* Add `AUTO` output format which switches from `ALIGNED` to `VERTICAL` if
+  the output doesn't fit the current terminal. ({issue}`12208`)
+## BigQuery connector
+
+* Add support for writing BigQuery `numeric` and `bignumeric` types. ({issue}`16145`)
```

Comment:
We prefer to talk about SQL support in terms of Trino types, so this should instead say - add support for writing `DECIMAL` types to BigQuery.

That is more important here because while we can INSERT into BIGNUMERIC columns in BigQuery a Trino user doesn't get to choose NUMERIC vs BIGQUERY. Everything written/created via Trino would be NUMERIC in BigQuery.